test/integration_test/main_test.go

```diff
@@ -35,91 +79,126 @@ func setup() error {
 	var operatorUpgradeHopsImages string
 	var logLevel string
 	var err error
+	var enableDash bool
+	var user, testBranch, testProduct, testType, testDescription, testTags string
+	var testsetID int
 
 	flag.StringVar(&ci_utils.PxDockerUsername,
-		"portworx-docker-username",
+		dockerUsernameFlag,
```

Comment:
why are we changing these flags? The existing tests will fail.

Reply:
> why are we changing these flags? The existing tests will fail.

If you look at the commit message it says clearly that these are moved to variables for easy code maintenance and readability.